tools/expotools/src/commands/UpdateVendoredModule.ts

```diff
@@ -678,6 +701,29 @@ async function action(options: ActionOptions) {
     return bundledNativeModules;
   });
 
+  const { name, version } = (await JsonFile.readAsync(path.join(tmpDir, 'package.json'))) as {
+    name: string;
+    version: string;
+  };
+  const semverPrefix = (options.semverPrefix != null ? options.semverPrefix : moduleConfig.semverPrefix) || '';
+
+  console.log(
+    `\nUpdating ${chalk.green(name)} in workspace projects...`
+  );
+ 
+  let needToRunYarn = false;
+  for (const path of await getPackageJsonPathsAsync()) {
+    const wasUpdated = await updateDependencyAsync(path, name, `${semverPrefix}${version}`);
+    needToRunYarn = needToRunYarn || wasUpdated;
+  }
```

Comment:
Sorry for nitpicking and for disliking `let` variables, but maybe sth like this would be more clear:

```javascript
async function updateWorkspaceDependencies(name: string, versionRange: string): Promise<boolean> {
  const paths = await getPackageJsonPathsAsync();
  const results = await Promise.all(paths.map(path => updateDependencyAsync(path, name, versionRange)));
  return results.some(Boolean);
}
```

and then

```javascript
const needToRunYarn = await updateWorkspaceDependencies(name, versionRange);
```